src/ftest.jl

```diff
@@ -0,0 +1,155 @@
+type FTestResult{N}
+    ssr::NTuple{N, Float64}
+    nparams::NTuple{N, Int}
```

Comment:
Probably better call this `dof`, as it's the result of calling `dof(model)`? Also `params` is going to have a slightly different meanings from `coef` (https://github.com/JuliaStats/StatsBase.jl/pull/274/).